requirements.txt

```diff
@@ -1,3 +1,3 @@
-ocrd >= 1.0.0b5
+ocrd >= 1.0.0b10
 click
-tesserocr >= 2.3.1
+tesserocr@git+https://github.com/sirfz/tesserocr#egg=tesserocr-2.4.0
```

Comment:
I could update https://pypi.org/project/ocrd-fork-tesserocr/ with a build of the current master of tesserocr. Using git links as dependency is somewhat brittle in my experience.

Reply:
Yes, I just noticed that `pip` does not always consider such releases as "newest", so it would not update over an existing 2.4.0 installation.

Using your PyPI release appears better to me, too. If you publish (under some version number as you like), I will change `requirements.txt` here accordingly.

Reply:
Published https://pypi.org/project/ocrd-fork-tesserocr/3.0.0rc2/ which includes https://github.com/sirfz/tesserocr/pull/185. It's only a source build, cannot get a wheel built but it should still work, albeit with compilation when pip installing it.

Reply:
done.